statusHandler/statusMetricsProvider.go

```diff
@@ -152,12 +152,26 @@ func (sm *statusMetrics) NetworkMetrics() map[string]interface{} {
 	currentRound := sm.loadUint64Metric(core.MetricCurrentRound)
 	roundNumberAtEpochStart := sm.loadUint64Metric(core.MetricRoundAtEpochStart)
 
-	networkMetrics[core.MetricNonce] = sm.loadUint64Metric(core.MetricNonce)
+	currentNonce := sm.loadUint64Metric(core.MetricNonce)
+	nonceAtEpochStart := sm.loadUint64Metric(core.MetricNonceAtEpochStart)
+
+	networkMetrics[core.MetricNonce] = currentNonce
 	networkMetrics[core.MetricCurrentRound] = currentRound
-	networkMetrics[core.MetricEpochNumber] = sm.loadUint64Metric(core.MetricEpochNumber)
 	networkMetrics[core.MetricRoundAtEpochStart] = roundNumberAtEpochStart
+	networkMetrics[core.MetricNonceAtEpochStart] = nonceAtEpochStart
+	networkMetrics[core.MetricEpochNumber] = sm.loadUint64Metric(core.MetricEpochNumber)
 	networkMetrics[core.MetricRoundsPerEpoch] = sm.loadUint64Metric(core.MetricRoundsPerEpoch)
-	networkMetrics[core.MetricRoundsPassedInCurrentEpoch] = currentRound - roundNumberAtEpochStart
+	roundsPassedInEpoch := uint64(0)
+	if currentRound >= roundNumberAtEpochStart {
+		roundsPassedInEpoch = currentRound - roundNumberAtEpochStart
+	}
+	networkMetrics[core.MetricRoundsPassedInCurrentEpoch] = roundsPassedInEpoch
+
+	noncesPassedInEpoch := uint64(0)
```

Comment:
Why can't the clients calculate this? They already have the other two metrics

Reply:
I know. But this was the requirement.